portal-api/src/v1/users/passport.js

```diff
@@ -34,6 +34,7 @@ const baseAuthenticationConfiguration = ({ name, passport, additionalValidation,
     new JwtStrategy(options, async (jwtPayload, done) => {
       findByUsername(jwtPayload.username, (error, user) => {
         if (error) {
+          console.error(`Failed when finding username '${jwtPayload.username}' in database during '${name}' JWT strategy`);
```

Comment:
Please use string substitution here and for the rest of the similar style console logs.

```javascript
 console.error('Failed when finding username %s in database during %s JWT strategy', jwtPayload.username, name);
```